src/screens/FreeProducts/index.tsx

```diff
@@ -0,0 +1,71 @@
+import './index.css'
+
+import React, { useState, useEffect } from 'react'
+import { Header } from 'src/components/UI'
+import { generateLink } from './utils/utils'
+import { createNewWindow } from 'src/helpers'
+import { useTranslation } from 'react-i18next'
+import { IpcRenderer } from 'electron'
+import { FreeGameElement } from 'src/types'
+const { ipcRenderer } = window.require('electron') as {
+    ipcRenderer: IpcRenderer
+}
+
+const FreeProducts = () => {
+  const { t, i18n } = useTranslation()
+  let lang = i18n.language
+  if (i18n.language === 'pt') {
```

Comment:
maybe this fix for the pt language can be moved inside the `generateLink` function? also, is this safe? can't this be `pt-PT` too for portugal?